Misc/NEWS.d/next/Core and Builtins/2019-01-05-18-39-49.bpo-35634.nVP_gs.rst

```diff
@@ -0,0 +1,3 @@
+``func(**kwargs)`` will now raise an error when ``kwargs`` is a mapping
+containing multiple entries with the same key. An error was already raised
+when other keyword arguments are passed before ``**kwargs`` since Python 3.6.
```

Comment:
The following cases are errors since 3.6:
```python
g(a=3, **MultiDict([('x', 1), ('x', 2)]))
g(**MultiDict([('a', 3)]), **MultiDict([('x', 1), ('x', 2)]))
```